arduino/i2c_slave_code_rccar/i2c_slave_rccar/i2c_slave_rccar.ino

```diff
@@ -69,6 +75,10 @@ bool checkRemote() {
     // read PWM from receiver
     remoteControlSpeed = pulseIn(THROTTLE, HIGH);
     remoteControlSteering = pulseIn(STEERING, HIGH);
+    
+    
+    
+    Serial.println("speed " + String(remoteControlSpeed) + " steering " +  String(remoteControlSteering));
```

Comment:
are you reading this anywhere? If not you should at least comment it out as that's going to really slow down your main loop.